doc/devel/testing.rst

```diff
@@ -53,15 +56,6 @@ not need to be installed, but Matplotlib should be)::
 
   pytest lib/matplotlib/tests/test_simplification.py::test_clipping
 
-or, if tests are installed, a dot-separated path to the module, optionally
-followed by the function separated by two colons, such as::
-
-  pytest --pyargs matplotlib.tests.test_simplification::test_clipping
```

Comment:
There's no benefit in this alternative. It's more to type and you don't have tab-completion for the filename part.

Reply:
It seems losing this part loses the only place that mentions how to run tests on an installed copy.